src/west/cmd/project.py

```diff
@@ -225,6 +225,39 @@ def do_run(self, args, user_args):
                 _git(project, 'status', extra_args=user_args)
 
 
+class ForAll(WestCommand):
+    def __init__(self):
+        super().__init__(
+            'forall',
+            _wrap('''
+            Runs a shell (Linux) or batch (Windows) command within the
+            repository of each of the specified projects (default: all cloned
+            projects). Note that you have to quote the command if it consists
```

Comment:
FWIW, I think requiring the quoting is a fine thing. Much less ambiguity in the presence of multiple shells we need to support.